packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposeOffchainConfig.ts

```diff
@@ -98,6 +107,29 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
   }
 }
 
+const beforeExecute: BeforeExecute<CommandInput, ContractInput> = (context) => async () => {
+  const event = await getLatestOCRConfig(context.provider, context.contract)
+  const offchainConfig = event?.offchain_config
+    ? await deserializeConfig(Buffer.from(event.offchain_config[0], 'base64'))
+    : ({} as OffchainConfig)
+
+  const contractOffchainConfig = {
+    ...offchainConfig,
```

Comment:
Let's just compare `offchainConfig`

Reply:
It's basically just `offchainConfig` there. The rest are transformations for the sake of comparison. e.g. `offchainPublicKeys` is part of offchainConfig but it's coming as raw Buffer from the contract while rdd has those keys hex encoded